app/src/main/java/org/p2p/wallet/striga/user/api/StrigaApi.kt

```diff
@@ -9,6 +9,19 @@ interface StrigaApi {
     @POST("v1/user/{userId}")
     suspend fun getUserDetails(@Path("userId") userId: String): StrigaUserDetailsResponse
 
+    /**
+     * 30044 - mobile already in use
+     * 30031 - invalid verification code
+     * 30003 - exceeded verification attempts
+     * 30005 - user doesn't exist, critical error
+     */
     @POST("v1/user/verify-mobile")
     suspend fun verifyMobileNumber(@Body body: StrigaVerifyMobileNumberRequest)
+
+    /**
+     * 31009 - mobile already in use
```

Comment:
we have StrigaApiErrorCode to handle this errors
I thought it would be better for you as a users of this api to know what to catch